be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -162,74 +161,90 @@ Status ExchangeSinkOperator::Channel::init(RuntimeState* state) {
     return Status::OK();
 }
 
-Status ExchangeSinkOperator::Channel::add_rows_selective(vectorized::Chunk* chunk, const uint32_t* indexes,
-                                                         uint32_t from, uint32_t size, RuntimeState* state) {
-    if (UNLIKELY(_chunk == nullptr)) {
-        _chunk = chunk->clone_empty_with_tuple();
+Status ExchangeSinkOperator::Channel::add_rows_selective(vectorized::Chunk* chunk, int32_t shuffle_id,
+                                                         const uint32_t* indexes, uint32_t from, uint32_t size,
+                                                         RuntimeState* state) {
+    if (UNLIKELY(_chunks[shuffle_id] == nullptr)) {
+        _chunks[shuffle_id] = chunk->clone_empty_with_tuple();
```

Comment:
clone_empty_with_slot is enough. pipeline engine no tuple id columns

Reply:
ok